integration/default_set.yaml

```
@@ -376,3 +398,34 @@ tests:
       - command: import -b zeek
         input: data/zeek/conn.log.gz
       - command: count '#type ~ /suricata.*/'
+  # The idea is import just enough to hit the high-water mark, wait, and then
+  # verify that most of the partitions and segments were deleted by checking
+  # the number of events left in the database. This test is a bit annoying as
+  # the numbers of the fixture must be set up quite precisely to trigger the
+  # disk monitor at the last import, but on the other hand it will also allow
+  # us to catch changes that drastically change the space used by the on-disk
+  # layout of the VAST data structures.
+  Disk monitor:
+    tags: [disk-quota]
+    fixture: DiskMonitorTester
+    # Disable on MacOS, since the additional `.DS_store` files throw off the size
+    # calculation.
+    condition: version | uname -s | grep Linux
```

```suggestion
    condition: version | uname -s | grep -v Darwin
```

This matches the above comment better.